_pages/attributes.md

```diff
@@ -1,26 +1,23 @@
-| ---------------------- | ------------  | ------------------------ | ----- | ----- |
-| UUID                   | `uuid`        | `sub`                    | <img src="{{ site.baseurl}}/assets/img/check.svg" alt="checkmark"> | <img src="{{ site.baseurl}}/assets/img/check.svg" alt="checkmark"> |
-| Email                  | `email`       | `email`                  | <img src="{{ site.baseurl}}/assets/img/check.svg" alt="checkmark"> | <img src="{{ site.baseurl}}/assets/img/check.svg" alt="checkmark"> |
-| First name             | `first_name`  | `given_name`             |       | <img src="{{ site.baseurl}}/assets/img/check.svg" alt="checkmark"> |
-| Middle name            | `middle_name` |                          |       | <img src="{{ site.baseurl}}/assets/img/check.svg" alt="checkmark"> |
-| Last name              | `last_name`   | `family_name`            |       | <img src="{{ site.baseurl}}/assets/img/check.svg" alt="checkmark"> |
-| Address line 1         | `address1`    | `address.street_address` |       | <img src="{{ site.baseurl}}/assets/img/check.svg" alt="checkmark"> |
-| Address line 2         | `address2`    | `address.street_address` |       | <img src="{{ site.baseurl}}/assets/img/check.svg" alt="checkmark"> |
-| City                   | `city`        | `address.locality`       |       | <img src="{{ site.baseurl}}/assets/img/check.svg" alt="checkmark"> |
-| State                  | `state`       | `address.region`         |       | <img src="{{ site.baseurl}}/assets/img/check.svg" alt="checkmark"> |
-| Zip code               | `zipcode`     | `address.postal_code`    |       | <img src="{{ site.baseurl}}/assets/img/check.svg" alt="checkmark"> |
-| Date of birth          | `dob`         | `birthdate`              |       | <img src="{{ site.baseurl}}/assets/img/check.svg" alt="checkmark"> |
-| Social security number | `ssn`         | `social_security_number` |       | <img src="{{ site.baseurl}}/assets/img/check.svg" alt="checkmark"> |
-| Phone                  | `phone`       | `phone`                  |       | <img src="{{ site.baseurl}}/assets/img/check.svg" alt="checkmark"> |
+| Attribute              | LOA1 | LOA3  | OpenID Connect | SAML |
+| ---------------------- | ---- | ----- | -------------- | ---- |
+| [UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier) | <img src="{{ site.baseurl }}/assets/img/check.svg" alt="checkmark"> | <img src="{{ site.baseurl }}/assets/img/check.svg" alt="checkmark"> | `sub` | `uuid` |
+| Email                  | <img src="{{ site.baseurl }}/assets/img/check.svg" alt="checkmark"> | <img src="{{ site.baseurl }}/assets/img/check.svg" alt="checkmark"> | `email` | `email` |
+| First name             |      | <img src="{{ site.baseurl }}/assets/img/check.svg" alt="checkmark"> | `given_name`             | `first_name`  |
+| Middle name            |      | <img src="{{ site.baseurl }}/assets/img/check.svg" alt="checkmark"> |                          | `middle_name` |
```

Comment:
I don't think we capture middle name anywhere

Reply:
Oh, how about that. Thanks @andrewhughey!